### DISCUSSION TITLE
Add a literal for Invariant culture string interpolation

### DISCUSSION THREAD

**Discussion by satikcz** [dotnet/csharplang#7974] [Language Ideas]
Add a literal for Invariant culture string interpolation so writing interpolated string with invariant culture is as straightforward as writing interpolated string with current culture.

Something like
`config.Add($I"{entry.Key}={entry.Value}")`
In the sample I is used as literal that denotes invariant culture interpolated string but that can be literally any available character.

**Comment by jnm2** [comment]
~~Today's solution~~ Solution prior to C# 10:

```cs
config.Add(FormattableString.Invariant($"{entry.Key}={entry.Value}"));
```

**Reply by 333fred** [reply]
Actually, it's `string.Create(CultureInfo.InvariantCulture, $"{entry.Key}={entry.Value}")`. FormattableString is the pre-C# 10 solution.
